source/opt/basic_block.h

```diff
@@ -98,6 +100,12 @@ class BasicBlock {
     return --insts_.cend();
   }
 
+  // Return true if the basic block has at least one successor
```

Comment:
Returns and missing period.[Function comments](https://google.github.io/styleguide/cppguide.html#Function_Comments)